src/ajitSJune20/FindMaxValAtDifferIndex.java

```diff
@@ -0,0 +1,33 @@
+/*Program 4: Find max value at differed index.
+              i/p:- arr1: {10,2,9,14,3}
+                    arr2: {10,2,18,14,3}
+              o/p :- Values are not matching at index -> 2, From (9,18) max value is 18*/
+
+package ajitSJune20;
+
+public class FindMaxValAtDifferIndex {
+	int index= 0;
+	void maxValAtIndex(int[] array1, int[] array2) {
+		if (array1.length==array2.length) {
+			for (int index= 0; index<array2.length; index++) {
+				if(array1[index]!=array2[index]) {
+					int maxVal=0;
```

Comment:
If you have initialized int maxVal=array2[index]; then no need to write else block here.